src/frontends/paddle/src/op/set_value.cpp

```diff
@@ -0,0 +1,180 @@
+// // Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+#define INT_MAX 2147483647
```

Comment:
please, remove and use https://en.cppreference.com/w/c/types/limits

Reply:
I will modify this by next PR

Reply:
Already fixed here https://github.com/openvinotoolkit/openvino/pull/17890